presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcConfig.java

```diff
@@ -99,4 +105,46 @@ public BaseJdbcConfig setUnsupportedTypeHandling(UnsupportedTypeHandling unsuppo
         this.unsupportedTypeHandling = unsupportedTypeHandling;
         return this;
     }
+
+    public boolean isUnspecifiedDecimalEnabled()
+    {
+        return unspecifiedDecimalEnabled;
+    }
+
+    @Config("unspecified-decimal-enabled")
+    @ConfigDescription("Enable unspecified decimals population")
```

Comment:
I would remove `population` word. @mosabua What do you think?
Maybe use `mapping` instead